src/lib/State/blocks.js

```diff
@@ -4,15 +4,15 @@ const { datesGenerator } = require('../../../backend/lib/mes-aides');
 const { Step, ComplexStep } = require('./steps');
 
 function individuBlockFactory(id) {
-  const r = variable => new Step({entity: 'individu', id, variable})
+  const r = (variable, chapter) => new Step({entity: 'individu', id, variable, chapter})
   const conjoint = id == 'conjoint'
   const demandeur = id == 'demandeur'
   const enfant = id.startsWith('enfant')
   return {
     subject: situation => situation[id] || situation.enfants.find(enfant => enfant.id === id) || {},
     steps: [
       ...(enfant ? [r('_firstName')] : []),
-      r('date_naissance'),
+      r('date_naissance', demandeur ? 'profil' : 'foyer'),
```

Comment:
```suggestion
      r('date_naissance', demandeur ? 'profil' : undefined),
```

Me parait plus adapté car il permet d'éviter de créer un nouveau chapitre pour chaque individu.